jetty-core/jetty-rewrite/src/main/java/org/eclipse/jetty/rewrite/handler/RegexRule.java

```diff
@@ -69,7 +69,7 @@ public Processor matchAndApply(Processor input) throws IOException
      * @return the possibly wrapped {@code Request} and {@code Processor}
      * @throws IOException if applying the rule failed
      */
-    protected abstract Processor apply(Processor input, Matcher matcher) throws IOException;
```

Comment:
Change line 69 to:
`     * @return the possibly wrapped {@code Request} and {@code Handler}`

Or use `{@link ...}` instead of `{@code ...}`?